client/renderers/EGL/filter_downscale.c

```diff
@@ -77,12 +77,74 @@ static void egl_filterDownscaleEarlyInit(void)
   // as it's per title. We need presets to make this nicer to use.
   static struct Option options[] =
   {
+    {
+      .module        = "eglFilter",
+      .name          = "downscale",
+      .description   = "Enable downscaling",
+      .type          = OPTION_TYPE_BOOL,
+      .value.x_bool  = false
+    },
+    {
+      .module        = "eglFilter",
+      .name          = "downscalePixelSize",
+      .description   = "Downscale filter pixel size",
+      .type          = OPTION_TYPE_FLOAT,
+      .value.x_float = 2.0f
+    },
+    {
+      .module        = "eglFilter",
+      .name          = "downscaleHOffset",
+      .description   = "Downscale filter horizontal offset",
+      .type          = OPTION_TYPE_FLOAT,
+      .value.x_float = 0.0f
+    },
+    {
+      .module        = "eglFilter",
+      .name          = "downscaleVOffset",
+      .description   = "Downscale filter vertical offset",
+      .type          = OPTION_TYPE_FLOAT,
+      .value.x_float = 0.0f
+    },
+    {
+      .module        = "eglFilter",
+      .name          = "downscaleFilter",
+      .description   = "Downscale filter type",
+      .type          = OPTION_TYPE_INT,
+      .value.x_int   = 0
+    },
     { 0 }
   };
 
   option_register(options);
```

Comment:
Should we even be registering these as command-line options at all? This is going to get very messy if we do this as more filters/options are added.